sql/src/main/java/org/apache/druid/sql/DirectStatement.java

```diff
@@ -245,6 +240,17 @@ public ResultSet plan()
     }
   }
 
+  protected DruidPlanner getPlanner()
```

Comment:
nit: `createPlanner` is a better name; `getPlanner` suggests it's getting a single thing (but this method creates a new planner each time).

Reply:
done